velox/dwio/parquet/tests/ParquetWriterTestBase.h

```diff
@@ -20,32 +20,29 @@
 #include <string>
 #include "velox/common/base/Fs.h"
 #include "velox/dwio/common/FileSink.h"
-#include "velox/dwio/common/tests/utils/BatchMaker.h"
 #include "velox/dwio/common/tests/utils/DataFiles.h"
 #include "velox/dwio/parquet/writer/Writer.h"
 #include "velox/exec/tests/utils/TempDirectoryPath.h"
-#include "velox/vector/tests/utils/VectorMaker.h"
+#include "velox/vector/fuzzer/VectorFuzzer.h"
 
 namespace facebook::velox::parquet {
 
 class ParquetWriterTestBase : public testing::Test {
  protected:
-  void SetUp() override {
+  static void SetUpTestSuite() {
     dwio::common::LocalFileSink::registerFactory();
     rootPool_ = memory::defaultMemoryManager().addRootPool("ParquetWriterTest");
     leafPool_ = rootPool_->addLeafChild("SinkTest");
     tempPath_ = exec::test::TempDirectoryPath::create();
   }
 
-  std::vector<RowVectorPtr> createBatches(
-      const std::shared_ptr<const Type>& rowType,
-      uint64_t batchNum,
-      uint64_t capacity) {
+  std::vector<RowVectorPtr>
+  createBatches(uint64_t batchNum, uint64_t vectorSize, RowTypePtr rowType) {
```

Comment:
Would you please move the implementation into .cpp time to speed up incremental builds?